android/res/values/strings.xml

```diff
@@ -21,7 +21,16 @@
 <resources>
     <!-- Important: Android is buggy with HTML, do not put HTML tags right next to text, leave a space, as it seems the text wrapping logic is considering HTML as visible text and you get premature wrapping. -->
     <string name="your_privacy_is_a_fundamental_right_html">Your privacy is a fundamental human right.</string>
-    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[<p> * Although highly recommended, <b>you do not need a VPN to use FrostWire. </b></p>]]></string>
+    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[* Although recommended for your downloads, <b>you do not need a VPN to use FrostWire. </b>]]></string>
```

Comment:
Yes. Per suggestions on improving conversion of the VPN purchases - the company recommended to include suggestion for their service when a user is already using a VPN provider. 

If I would like to keep the formatting than, what is the best way to approach it?